pkg/controllers/dynakube/oneagent/daemonset/arguments.go

```diff
@@ -85,8 +85,7 @@ func (b *builder) appendNoProxyArg(argMap *prioritymap.Map) {
 		noProxyValue := b.dk.FF().GetNoProxy()
 
 		if b.dk.ActiveGate().IsEnabled() {
-			multiCap := capability.NewMultiCapability(b.dk)
-			noProxyActiveGateValue := capability.BuildDNSEntryPointWithoutEnvVars(b.dk.Name, b.dk.Namespace, multiCap)
+			noProxyActiveGateValue := capability.BuildHostEntries(*b.dk)
```

Comment:
I suppose AG should be ALWAYS added to the no-proxy list. It depends on a custom ff now.

OS agent
```
2025-04-23 12:11:18.542 UTC [00049826] info    [comm  ] URL [https://3...:443/communication](https://3.../communication) not working (Could not resolve proxy: squid.proxy.svc.cluster.local) 
2025-04-23 12:11:18.548 UTC [00049826] info    [comm  ] URL [https://dynakube-activegate.dynatrace:443/communication](https://dynakube-activegate.dynatrace/communication) not working (Could not resolve proxy: squid.proxy.svc.cluster.local)
```